packages/client3/components/design-system/file-upload/types/file-upload.types.ts

```diff
@@ -0,0 +1,7 @@
+export interface FileUploadProps {
+  id: string;
+  label: string;
+  name: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: string;
+}
```

Comment:
Is this file needed as I think the fileupload props are already declared within file

Reply:
Perfect, added that just in case we need to doooo...something?